framework/src/timeintegrators/TimeIntegrator.C

```diff
@@ -69,3 +70,22 @@ TimeIntegrator::getNumLinearIterationsLastSolve() const
 
   return nonlinear_solver.get_total_linear_iterations();
 }
+
+void
+TimeIntegrator::computeDotProperties(Moose::MaterialDataType type, THREAD_ID tid) const
+{
+  auto material_data = _fe_problem.getMaterialData(type, tid);
+
+  auto props = material_data->getPropertiesForDot();
+  auto old_props = material_data->getOldPropertiesForDot();
+  auto dot_props = material_data->getDotProperties();
```

Comment:
😱 those `auto`s should be `auto &` (or `const auto &` if you can).  You are making copies of these vectors here!  Be careful!